src/main/api/ledger/transaction.ts

```diff
@@ -1,5 +1,4 @@
-import Transport from '@ledgerhq/hw-transport'
-import TransportNodeHidSingleton from '@ledgerhq/hw-transport-node-hid-singleton'
+import * as Transport from '@ledgerhq/hw-transport'
```

Comment:
```suggestion
import { default as Transport } from '@ledgerhq/hw-transport'
```

There's no reason to spread `.default` to every use site in the code base like this, right?

Reply:
Please make a PR for this post merge.